### DISCUSSION TITLE
Can't see who user follows in `started following N users` feed event

### DISCUSSION THREAD

**Discussion by MaxLeiter** [community/community#127752] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

![CleanShot 2024-06-07 at 15 06 02@2x](https://github.com/community/community/assets/8675906/edaaa1f2-cf18-4bfe-a7ed-6ed032be6d2b)

When this event comes up, I can't see who the person is following that makes it relevant to me. It's just a random profile.

**Comment by Lancern** [comment]
The message is really confusing. It should actually be "XXX and other N users started following you".
